network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt

```diff
@@ -81,13 +81,6 @@ enum class NetworkProtectionPixelNames(
     NETP_TERMS_SHOWN_DAILY("m_netp_imp_terms_d"),
```

Comment:
All `NETP_TERMS_` can also be removed now since we don’t have the terms screen anymore

Reply:
will remove